sos/utilities.py

```diff
@@ -210,6 +210,45 @@ def get_human_readable(size, precision=2):
         size = size/1024.0
     return "%.*f%s" % (precision, size, suffixes[suffixindex])
 
+def get_network_namespaces(namespace_pattern = None, namespaces_max = 0,
+                           log = None):
+    """Used to get a list of network namespaces, optionally filtered
+    by a regex pattern or max number of namespaces (options from the
+    networking plugin.)
+    If you specify max number of namespaces (namespaces_max) specify
+    a log function too, to log if the limit was reached.
```

Comment:
Nack. Use the logging already setup by sos. This will be available inside the `Plugin` class already.

Reply:
Face palm. I missed that came from `Plugin`. Fixed it.